### PR TITLE
Feature.51823

### PR DESCRIPTION
**Pull request by dangthibichngocb**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/51823)

## WHAT (optional)
- crud category in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
Screenshot from 2022-08-15 21-55-32

### CODE REVIEW

**Comment by dangthibichngocb** [comment]
ready

**Comment by dangthibichngocb** [comment]
ready

File: app/views/admin/categories/_category.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      <%= category.status_i18n %>
+    </td>
+    <td class="w-25">
+      <%= link_to edit_admin_category_path(category), class: "btn btn-default float-left" do %>
+        <i class=" fas fa-edit" aria-hidden="true"></i>
+        <% end %>
+          <%= button_to admin_category_path(category), remote: true, method: :delete,
+            class: "btn btn-danger float-right" do%>
+            <i class="fas fa-times" aria-hidden="true"></i>
+            <% end %>
```
Comment (ghost) [inline app/views/admin/categories/_category.html.erb]: sai indent

File: app/views/admin/categories/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  <%= form_for [:admin, @category] do |f| %>
+    <div class="col-md-6">
+      <div class="card card-primary">
+        <div class="card-header">
+        </div>
+        <form>
+          <div class="card-body">
+            <div class="form-group">
+              <%= f.label t(".name"), class:"control-label" %>
+                <%= f.text_field :name, class: "form-control" %>
```
Comment (ghost) [inline app/views/admin/categories/_form.html.erb]: indent

**Review by ghost — COMMENTED** [review]
self review lại em nhé

**Comment by dangthibichngocb** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**